bookdown/02-model-optimization/01-tuning.Rmd

```diff
@@ -1,5 +1,7 @@
 ## Hyperparameter Tuning {#tuning}
 
+Hyperparameter tuning differentiates from parameter tuning as parameters are set prior to training. 
```

Comment:
Hm, I am not sure if this sentence is really good as the first one here. Hyperpars can also be set "before" the training. I know what you mean but this is not really clear here.

Reply:
Good point! I will look up a more suited definition.